doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,26 @@
+Electron Holography
+*******************
+
+HyperSpy provides the user with two classes which can be used to process electron holography data:
+
+* :py:class:`~._signals.hologram_image.HologramImage`
+* :py:class:`~._signals.wave_image.WaveImage`
+
+Both inherit directly from the :py:class:`~._signals.signal2d.Signal2D` class and thus can use all of its
+functionality. The usage of both classes is explained in the following sections.
+
+
+
+The WaveImage class
+===================
+
+The :py:class:`~._signals.wave_image.WaveImage` class can hold information about the complex electron

```

Comment:
I bet that a lot of the functionality of `WaveImage` may also be useful in other domains beyond holography. Is there any exception? If yes, would it make sense to create an `ElectronWaveImage` class? The objectives are i) keep the objects and documentation as general as possible (HyperSpy is not only for EM) ii) avoid cluttering the objects with methods that are domain specific.